FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/AuthPickerView.swift

```diff
@@ -29,28 +26,32 @@ extension AuthPickerView: View {
       } else if authService.authView == .emailLink {
         EmailLinkView()
       } else {
-        Text(authService.authenticationFlow == .login ? authService.string
-          .emailLoginFlowLabel : authService.string.emailSignUpFlowLabel)
-        VStack { Divider() }
-        EmailAuthView()
+        if authService.emailSignInEnabled {
+          Text(authService.authenticationFlow == .login ? authService.string
+            .emailLoginFlowLabel : authService.string.emailSignUpFlowLabel)
+          VStack { Divider() }
```

Comment:
This is kind of lazy of me to ask, but why does the divider need to be in another VStack?

Reply:
That's a good catch - I got this from the example app Peter provided a while ago. I tested using Preview here: [`d13aea6` (#1252)](https://github.com/firebase/FirebaseUI-iOS/pull/1252/commits/d13aea63bf1e2c0c80af61820969f34a90b28fca)

and found that it made no discernible difference to the UI so I've removed them.